package.json

```diff
@@ -2,6 +2,15 @@
   "name": "broadcast-channel",
   "version": "4.0.0",
   "description": "A BroadcastChannel that works in New Browsers, Old Browsers, WebWorkers and NodeJs",
+  "exports": {
+    "node": {
+      "import": "./dist/es/index.js",
+      "default": "./dist/es5node/index.js"
+    },
+    "browser": {
+      "default": "./dist/lib/index.es5.js"
```

Comment:
yeah, I deleted my suggestion because I realized that would be an issue. I hadn't seen you'd already replied to it :smile: I think we could consider that as a follow up, but this PR looks good to me as is for now